.pre-commit-config.yaml

```diff
@@ -1,12 +1,7 @@
 exclude: "src/scantree/_version.py"
 repos:
-  - repo: https://github.com/pre-commit/mirrors-prettier
```

Comment:
removed mirrors-prettier as the repo got deprecated (underlying prettier changes breaking the project altogether)